contracts/vaults/STETHVaultWithSwap/base/RibbonVault.sol

```diff
@@ -0,0 +1,795 @@
+// SPDX-License-Identifier: MIT
+pragma solidity =0.8.4;
```

Comment:
My suggestion would be to just implement the changes into RibbonSTETHVault directly, rather than add a new RibbonThetaSTETHVaultWithSwap. This is because we won't be regressing back into Gnosis auctions anymore and it's easier to view the diff

Reply:
Same for the Yearn contracts

Reply:
Do we need to do it for yearn contracts? Since we will be reverting YV to base theta vault